migrations/postgres/1_create_db_init.down.sql

```diff
@@ -0,0 +1,23 @@
+-- MIT License
+--
+-- (C) Copyright [2019-2021] Hewlett Packard Enterprise Development LP
```

Comment:
Should the copyright be changed to reflect the license used for OChami?

Reply:
Resolved in [80f702c](https://github.com/OpenCHAMI/bss/pull/13/commits/80f702ce021c50f3e44003ab18574dcf17bb9078).